js/main.js

```diff
@@ -0,0 +1,55 @@
+
+const commentMassage = [
```

Comment:
Массивы надо называть во множественном числе, так будет понятнее что мы имеем дело с коллекцией
```suggestion
const COMMENT_MESSAGES = [
```